docs/reference/ml/df-analytics/apis/evaluate-dfanalytics.asciidoc

```diff
@@ -246,4 +249,85 @@ only. It means that a testing error will be calculated.
 <2> The field that contains the ground truth value for the actual student 
 performance. This is required in order to evaluate results.
 <3> The field that contains the predicted value for student performance 
-calculated by the {reganalysis}.
\ No newline at end of file
+calculated by the {reganalysis}.
+
+
+===== {classification-cap}
+
+
+[source,console]
+--------------------------------------------------
+POST _ml/data_frame/_evaluate
+{ 
+   "index": "animal_classification",
+   "evaluation": {
+      "classification": { <1>
+         "actual_field": "animal_class", <2>
+         "predicted_field": "ml.animal_class_prediction.keyword", <3>
+         "metrics": {  
+           "multiclass_confusion_matrix" : {}
+         }
+      }
+   }
+}
+--------------------------------------------------
+// TEST[skip:TBD]
+
+<1> The evaluation type.
+<2> The field that contains the ground truth value for the actual animal 
+classification. This is required in order to evaluate results.
+<3> The field that contains the predicted value for animal classification by 
+the {classanalysis}. Note that you need to add the `.keyword` suffix to the 
```

Comment:
You may also explain why is this `.keyword` suffix needed. Sth like: "Since the field storing predicted class is dynamically mapped as text and keyword, you need to add ..."